src/pages/workspace/rules/RulesAutoPayReportsUnderPage.tsx

```diff
@@ -0,0 +1,92 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import React from 'react';
+import {View} from 'react-native';
+import AmountForm from '@components/AmountForm';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapper from '@components/Form/InputWrapper';
+import type {FormInputErrors, FormOnyxValues} from '@components/Form/types';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import ScreenWrapper from '@components/ScreenWrapper';
+import Text from '@components/Text';
+import useAutoFocusInput from '@hooks/useAutoFocusInput';
+import useLocalize from '@hooks/useLocalize';
+import usePolicy from '@hooks/usePolicy';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as CurrencyUtils from '@libs/CurrencyUtils';
+import Navigation from '@libs/Navigation/Navigation';
+import type {SettingsNavigatorParamList} from '@libs/Navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import * as PolicyActions from '@userActions/Policy/Policy';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
+import INPUT_IDS from '@src/types/form/RulesAutoPayReportsUnderModalForm';
+
+type RulesAutoPayReportsUnderPageProps = StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.RULES_AUTO_PAY_REPORTS_UNDER>;
+
+function RulesAutoPayReportsUnderPage({route}: RulesAutoPayReportsUnderPageProps) {
+    const {policyID} = route.params;
+    const policy = usePolicy(policyID);
+
+    const {inputCallbackRef} = useAutoFocusInput();
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+
+    const defaultValue = CurrencyUtils.convertToFrontendAmountAsString(policy?.autoReimbursement?.limit, policy?.outputCurrency);
+
+    const validateLimit = ({maxExpenseAutoPayAmount}: FormOnyxValues<typeof ONYXKEYS.FORMS.RULES_AUTO_PAY_REPORTS_UNDER_MODAL_FORM>) => {
+        const errors: FormInputErrors<typeof ONYXKEYS.FORMS.RULES_AUTO_PAY_REPORTS_UNDER_MODAL_FORM> = {};
+        if (CurrencyUtils.convertToBackendAmount(parseFloat(maxExpenseAutoPayAmount)) > CONST.POLICY.AUTO_REIMBURSEMENT_MAX_LIMIT_CENTS) {
+            errors[INPUT_IDS.MAX_EXPENSE_AUTO_PAY_AMOUNT] = translate('common.error.invalidAmount');
+        }
+        return errors;
+    };
```

Comment:
I think this should be applied in the USD, but I wasn't sure how exactly we should be handling this

Reply:
There's no special handling for currency with regards to this limit. Does that answer it?

Not sure I would expect people to be putting very large amounts into this field in any case since it's impractical to do so. What you have here seems fine.

Reply:
ok, but if someone has yen selected as a policy currency, this limit would be 20000 yen which is 139,02$. What I'm worried about is that because of the lack of currency handling in the validation, if the currency has a different denomination, the administrator might not be able to set a reliable limit.

Reply:
AFAICT we should not be converting this in the front end? Whatever it is in yen is what should be sent. So for your example of 20000 yen that would be: `2000000` sent to the API.